### PR TITLE
A PR for Issue #192 and #196

### PR DESCRIPTION
**Pull request by sinri**
I had done some small fixes for my own project, I wonder these could be some help for main repo.
By the advise of @devenbansod I made this PR. Please feel free to do with it.
THX.

### CODE REVIEW

**Comment by williamdes** [comment]
@sinri I fixed conflicts, I hope you are okay with that :)

File: src/Components/CaseExpression.php [hunk 1]
```diff
@@ -55,6 +56,14 @@ class CaseExpression extends Component
      */
     public $else_result;
 
+    /**
+     * The alias of this expression.
+     * Added by Sinri
```
Comment (devenbansod) [inline src/Components/CaseExpression.php]: @sinri no need to specify this. Could definitely get who wrote it via `git blame`

File: src/Components/Expression.php [hunk 2]
```diff
@@ -199,18 +200,24 @@ public static function parse(Parser $parser, TokensList $list, array $options =
              */
             $token = $list->tokens[$list->idx];
 
+            //echo __METHOD__ . '@' . __LINE__ . ' expression check token: ' . $token . PHP_EOL;
```
Comment (devenbansod) [inline src/Components/Expression.php]: @sinri I'm not sure if this file has any changes except for the commented-out debug statements.

**Comment by devenbansod** [comment]
Actually both of the targeted issues have been fixed. So, closing this PR.

File: src/Components/CaseExpression.php (hunk 1 above)
Comment (sinri) [reply src/Components/CaseExpression.php]: Well, in my own branch I wrote them to tell myself where I changed...
Remove it, good.

File: src/Components/Expression.php (hunk 2 above)
Comment (sinri) [reply src/Components/Expression.php]: So that should be just a debug message and no need for production environment.
Remove it, good.

**Comment by sinri** [comment]
Thank you @devenbansod

**Pull request closed without merging**